exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,16 @@
 
 let name = "Daniel";
 let danielsRole = "mentor";
+    
+function roleCheck (danielsRole) {
+if (danielsRole === "mentor") {
+  console.log("Hi, I am " + name + "," + "I am a " + danielsRole + ".");
+} else if (danielsRole === "student") {
+  console.log("Hi, I am " + name + "," + "I am a " + danielsRole+ ".");
```

Comment:
Hm, this console.log statement is **exactly** the same as when role is "mentor". Can simplify the function to reduce code duplication? :)